packages/services/api/src/modules/auth/lib/authz.ts

```diff
@@ -350,6 +350,7 @@ const actionDefinitions = {
   'target:modifySettings': defaultTargetIdentity,
   'laboratory:describe': defaultTargetIdentity,
   'laboratory:modify': defaultTargetIdentity,
+  'laboratory:modifyPreflightScript': defaultTargetIdentity,
```

Comment:
TODO @dimaMachina Grant this permission to `TargetAccessScope.SETTINGS` here: https://github.com/graphql-hive/console/blob/5838e821fc86d2f0a44003af775ceec2f05a4acd/packages/services/api/src/modules/auth/lib/supertokens-strategy.ts#L314

Reply:
granted